src/mpags-cipher.cpp

```diff
@@ -2,59 +2,39 @@
 #include <iostream>
 #include <string>
 #include <vector>
+#include <fstream>
+
+#include "TransformChar.hpp"
+#include "ProcessCommandLine.hpp"
+
+std::string transformChar( const char in_char );
+
+bool processCommandLine(const std::vector<std::string>& args,
+                        bool& helpRequested,
+                        bool& versionRequested,
+                        std::string& inputFile,
+                        std::string& outputFile,
+                        size_t& key,
+                        bool& encrypt);
+
+std::string runCaesarCipher(const std::string& inputText, const size_t key, const bool encrypt);
```

Comment:
Because you've included `TransformChar.hpp` and `ProcessCommandLine.hpp`, there is no need to write out the declarations here yourself - this is unnecessary and can easily lead to problems.  Similarly you should include `RunCaesarCipher.hpp` and drop the explicit declaration of that function as well.

```suggestion
#include "TransformChar.hpp"
#include "ProcessCommandLine.hpp"
#include "RunCaesarCipher.hpp"
```